server/plugin/client.go

```diff
@@ -236,6 +237,37 @@ func (c *client) GetGitRepositoryBranches(organization, projectName, repository,
 	return gitBranchesResponse, statusCode, nil
 }
 
+func (c *client) GetSubscriptionFilterPossibleValues(request *serializers.GetSubscriptionFilterPossibleValuesRequestPayload, mattermostUserID string) (*serializers.SubscriptionFilterPossibleValuesResponseFromClient, int, error) {
+	getSubscriptionFilterValuesURL := fmt.Sprintf(constants.GetSubscriptionFilterPossibleValues, request.Organization)
+
+	var subscriptionFilters []serializers.SubscriptionFilter
+	for _, filter := range request.Filters {
+		subscriptionFilters = append(subscriptionFilters, serializers.SubscriptionFilter{InputID: filter})
+	}
+
+	subscriptionFiltersRequest := &serializers.GetSubscriptionFilterValuesRequestPayloadFromClient{
+		Subscription: &serializers.CreateSubscriptionBodyPayload{
+			PublisherID:      "tfs",
+			ConsumerID:       "webHooks",
+			ConsumerActionID: "httpRequest",
+			EventType:        request.EventType,
+			PublisherInputs: serializers.PublisherInputsGeneric{
+				ProjectID: request.ProjectID,
+			},
+		},
+		InputValues: subscriptionFilters,
+		Scope:       10,
```

Comment:
What's this scope here? Shouldn't this be a constant variable?

Reply:
it is not going to be used anywhere else so, I think there is no need for a constant variable, I have added a comment for its role